docs/developing/oauth.rst

```diff
@@ -22,14 +24,22 @@ Redirect URIs
 
 Store the Client ID and Client Secret tokens for use with your application.
 
-* These are the recommended settings. For a better understanding of which settings you should choose, read this [introduction to OAuth](https://aaronparecki.com/oauth-2-simplified/) and this [guide to grant types](https://alexbilbie.com/guide-to-oauth-2-grants/)
+* These are the recommended settings. For a better understanding of which settings you should choose, read this `introduction to OAuth <https://aaronparecki.com/oauth-2-simplified/>`_ and this `guide to grant types <https://alexbilbie.com/guide-to-oauth-2-grants/>`_
 
 Requesting authorization
 ========================
 
 Inside your application, redirect to the OAuth authorization endpoint to receive an authorization code. The URL is https://ion.tjhsst.edu/oauth/authorize/
 
-To access the API, exchange this code for a (temporary) access token. The URL is https://ion.tjhsst.edu/oauth/token/
+To access the API, exchange this code for a (temporary) access token. The URL is https://ion.tjhsst.edu/oauth/token/.
+
+Python-social-auth
```

Comment:
```suggestion
python-social-auth
```

Reply:
This is a heading; its name should be capitalized.